lint.sh

```diff
@@ -6,7 +6,7 @@ function incr_err() {
 }
 
 OUTPUT=`git grep "patch(" | grep -v "autospec"`
```

Comment:
Can you add a comment before this section and the next saying something like `# Check for autospec in mock.patch calls` and `# Check migration names`?